llmfoundry/data/text_data.py

```diff
@@ -106,6 +106,7 @@ def __init__(
         self,
         tokenizer: PreTrainedTokenizerBase,
         max_seq_len: int,
+        token_encoding_type: str = 'int32',
```

Comment:
can we make default 'int64' so it doesn't break compatibility with old code

Reply:
I favor 64 too, at least for the current and coming release.

Reply:
resolved